app/(admin)/admin/notifications/page.test.tsx

```diff
@@ -1,15 +1,63 @@
 // /Users/ryanfurrer/Developer/GitHub/gridiron-survivor/app/(admin)/admin/notifications/page.test.tsx
 
-import { render, screen } from '@testing-library/react';
 import AdminNotifications from './page';
+import { getCurrentLeague } from '@/api/apiFunctions';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
 import React from 'react';
+import { sendEmailNotifications } from './actions/sendEmailNotification';
+
+let contentInput: HTMLInputElement,
+  emailButton: HTMLElement,
+  emailTestersButton: HTMLElement,
+  subjectInput: HTMLInputElement;
+
+jest.mock('@/api/apiFunctions', () => ({
+  getCurrentLeague: jest.fn(),
+}));
+
+jest.mock('./actions/sendEmailNotification', () => ({
+  sendEmailNotifications: jest.fn(),
+}));
 
 describe('Admin notifications page', () => {
-  it(`should render it's content`, () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+
     render(<AdminNotifications />);
+
+    contentInput = screen.getByTestId('content-text');
+    emailButton = screen.getByTestId('send-email');
+    emailTestersButton = screen.getByTestId('email-testers');
+    subjectInput = screen.getByTestId('subject-text');
+  });
+  it(`should render it's content`, () => {
```

Comment:
`should render it's content` it's here is unclear, is there a way we can reword this to make it clearer?

Reply:
This wasn't my test. I just copy had to add one thing to it. I think it's all green because I deleted it and repasted it